### DISCUSSION TITLE
sensor.inverter_load_ups_*  values

### DISCUSSION THREAD

**Discussion by sofkaski** [davidrapan/ha-solarman#24] [Q&A]
Have anybody else noticed and have any explanation for a bit weird behavior of UPS load values?
My setup is using Deye LV 3-Phase Hybrid Inverter. Inverter definition file is `deye_sg04lp3.yaml`. 

I have in my setup nothing connected to L3 (sensor.inverter_load_ups_l3_power), but yet it shows some power consumption.
I think L1 and L2 are showing about correct values where I have some load. 

There is nothing wrong in the integration since the value really comes from the inverter. I was just wondering if anybody has some idea why that happens? Inverter software bug?

**Comment by davidrapan** [comment]
Yeah you are right about that. I was wondering the same thing. It is also strange that in some cases (e.g when I'm charging battery from the grid) those values doubles +-. More so that individual L1 L2 L3 sensors does not show values under 10. When it should show under 10 value it just simply gives 0.

Great question, my inverter is connected even through UPS output and then through switch which activates when there is power outage so I was thinking it's cause of that. But if you don't have it even connected then this is proly false and I really don't know...

I don't think it's a bug cause it shows somewhat very specific values... so it has to have them from somewhere... but I'm really not sure

**Reply by sofkaski** [reply]
Could it be that some inverter own consumption is included in UPS load?

**Reply by davidrapan** [reply]
I do calculate this value (together with losses) and it does not correspond though... And even my device specifications say that the consumption of the device is somewhere around ~80W.
